Common/cpp/Tools/RuntimeDecorator.h

```diff
@@ -107,4 +107,7 @@ inline bool RuntimeDecorator::isReactRuntime(jsi::Runtime &rt) {
   return false;
 }
 
+std::string parseValue(jsi::Runtime &rt, jsi::Value const &value);
+std::string parseComplexValue(jsi::Runtime &rt, jsi::Object const &value);
```

Comment:
Do we really need two functions where the second one is just for complex types? Perhaps it would be better to implement it somehow similarly to `makeShareableClone` (and `makeShareableCloneRecursive`). So we would have `stringifyValue` which serves as an entry point and calls `stringifyValueRecursively` which, as the name suggests, is called recursively.